### DISCUSSION TITLE
uninstall pipx

### DISCUSSION THREAD

**Discussion by DylanBlockchainDev** [smartcontractkit/full-blockchain-solidity-course-py#1829] [Q&A] [answered]
how do I uninstall pipx?
I am using ubuntu.

**Comment by cromewar** [comment, marked as answer]
Hello @DylanBlockchainDev 

To completely uninstall pipx on Ubuntu, you can use the following command:

`sudo apt-get remove --purge pipx`

This will remove pipx and all its associated files.

Then you can use the following command to remove any remaining pipx files

`pipx uninstall-all`

This will remove all packages installed by pipx.

You can also delete the pipx directory, if you have installed pipx to a custom location.

`rm -rf ~/.local/pipx`

It is also recommended to check your environment variables for any pipx related variables and remove them.
